src/cryptonote_core/blockchain.cpp

```diff
@@ -2501,6 +2520,26 @@ bool Blockchain::check_tx_inputs(transaction& tx, tx_verification_context &tvc,
     }
   }
 
+  // from v7, sorted ins
+  if (hf_version >= 7) {
```

Comment:
Why use the shorthand here but the full form above?

Reply:
The code near 2337 had been using it a few times, so I cached it in a local. The code near 2522 started by using it just once, then one added each time, so never got cached. There's no particular reason other than that.